### PR TITLE
BGDIINF_SB-2161: Fixed calls to non-existant error function

### PR DESCRIPTION
**Pull request by fkm**
Probably due to a change of the log function we have calls to `log.error` which doesn't exist.

This modifies the calls to work with the current implementation of the log function.

It also modifies the function's default handling to include/preserve the fist parameter passed to the function.

[Test link](https://web-mapviewer.dev.bgdi.ch/bugfix-jira-bgdiinf_sb-2161-log-function/index.html)

### CODE REVIEW

File: src/utils/logging.js [hunk 1]
```diff
@@ -1,28 +1,48 @@
...
             default:
-                console.log(...message)
+                console.log(level, ...message)
         }
     }
     /* eslint-enable  no-console */
 }
+
+/* eslint-disable no-console */
+/** Same as console.error but doesn't log in production. */
```
Comment (pakb) [inline src/utils/logging.js:40]: this is not meant to be here no?

File: src/utils/logging.js [hunk 2]
```diff
@@ -1,28 +1,48 @@
...
+log.error = (...message) => console.error(...message)
+/** Same as console.info but doesn't log in production. */
+log.info = (...message) => console.info(...message)
+/** Same as console.debug but doesn't log in production. */
+log.debug = (...message) => console.debug(...message)
```
Comment (pakb) [inline src/utils/logging.js:45]: Maybe we could call back to `log()` function here, and modify how we call `console.xxx` there. This way we only use console.log and other in the main `log()` function

Comment (fkm) [reply src/utils/logging.js]: Yeah. The shorthands were a bit ugly. I updated them to use our `log` function so they can respect the `DEBUG` constant.

File: src/utils/logging.js (hunk 1 above)
Comment (fkm) [reply src/utils/logging.js]: Makes sense. I changed it.

**Review by pakb — APPROVED** [review]

**Pull request merged**